docs/DeveloperGuide.md

```diff
@@ -437,9 +473,29 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * 2a. The date format is wrong
     * 2a1. HobbyList shows a message about the right format.
+  
+* 2b. There are multiple dates in the add command
+    * 2b1. HobbyList will save the last date typed in.
```

Comment:
![image](https://user-images.githubusercontent.com/97277891/199949880-6ffc2919-8975-431f-bf5c-0415be17721a.png)
I think the order of the use case extension is 1 -> 2 -> 2a -> 2a1, not 1 -> 2a -> 2a1. So the number should be 1a and 1b instead of 2a and 2b.